extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduWriter.java

```diff
@@ -16,12 +16,14 @@
   private final KuduOperations operations;
   private final String tableName;
   private final KuduSession session;
+  private final BatchHandler batchHandler;
 
 
   public KuduWriter(String tableName, KuduOperations operations) {
     this.tableName = tableName;
     this.operations = operations;
     this.session = operations.getSession();
+    this.batchHandler = operations.getBatchHandler();
```

Comment:
I think this BatchHandler needs to be created using `this.session`, rather than creating a new session inside of KuduOperations